sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/DataCollection/DocumentHelper.cs

```diff
@@ -170,6 +171,16 @@ internal static ExceptionDocument CreateException(string exceptionType, string e
             return exceptionDocumentIngress;
         }
 
+        internal static Models.Trace ConvertToTrace(LogRecord logRecord)
+        {
+            return new Models.Trace()
+            {
+                DocumentType = DocumentIngressDocumentType.Trace,
+                Message = logRecord.FormattedMessage ?? logRecord.Body, // TODO: MAY NEED TO BUILD THE FORMATTED MESSAGE IF NOT AVAILABLE
```

Comment:
`FormattedMessage`, if available, gives a string like "Hello World".
`Body` gives a string like "Hello {name}".

It seems that Application Insights would always build the fully formatted string. I don't think we should do that here. In a future PR, I think we could explore building a string that contains the template and the key-value-pairs. (ex: "Hello {name}. name: World").

I don't want to do this extra work until I have the unit test project up and working for this project. I propose leaving this as a TODO for now.